Source/WTF/WTF.xcodeproj/project.pbxproj

```diff
@@ -3887,7 +3887,7 @@
 			);
 			outputPaths = (
 				"$(BUILT_PRODUCTS_DIR)/$(PRIVATE_HEADERS_FOLDER_PATH)/WTF.json",
-				"$(DSTROOT)/$(PRIVATE_HEADERS_FOLDER_PATH)/WTF.json",
+				"$(DSTROOT)$(SYSTEM_PREFIX)/$(PRIVATE_HEADERS_FOLDER_PATH)/WTF.json",
```

Comment:
We've never used `SYSTEM_PREFIX` before, and it's an extremely ambiguous name. Is there any chance that something that we are using elsewhere would work?